cmd/openfish/middleware/authentication.go

```diff
@@ -62,42 +74,29 @@ func ValidateJWT(aud string) func(*fiber.Ctx) error {
 			return api.Unauthorized(err)
 		}
 
-		// Extract subject and email.
+		// Extract email.
 		email := payload.Claims["email"].(string)
-		subject := payload.Subject
-		role := entities.DefaultRole
-
-		// Fetch user from datastore if they exist, else
-		// create new user.
-		user, err := services.GetUserByEmail(email)
-		if err != nil {
-			services.CreateUser(email, role)
-		} else {
-			role = user.Role
-		}
-
-		// Add subject, email, and user role to ctx.Locals.
-		ctx.Locals("subject", subject)
 		ctx.Locals("email", email)
-		ctx.Locals("role", role)
+
+		// Fetch user from datastore if they exist.
+		user, _ := services.GetUserByEmail(email)
```

Comment:
Why did we get rid of the error handling?

Reply:
fixed